packages/rulesets/src/shared/functions/index.ts

```diff
@@ -1 +1,3 @@
-export { default as uniquenessTags } from './uniquenessTags';
+import { default as uniquenessTags } from './uniquenessTags';
+
+export { uniquenessTags };
```

Comment:
this results in a slightly smaller output, as no `__importDefault` function is declared